tools/cldr-rdf/src/main/java/org/unicode/cldr/rdf/QueryClient.java

```diff
@@ -37,29 +47,104 @@ public static final QueryClient getInstance() {
     public static final String PREFIX_YAGO = "http://dbpedia.org/class/yago/";
     public static final String abstractLang = "en"; // for now
     
-    public static final String getStringOrNull(final QuerySolution qs, final String k) {
+    public static final String getLiteralOrNull(final QuerySolution qs, final String k) {
         final Literal l = qs.getLiteral(k);
-        if(l == null) return null;
+        return getLiteralOrNull(l);
+    }
+    
+	private static String getLiteralOrNull(final Literal l) {
+		if(l == null) return null;
```

Comment:
formatting is off?

Reply:
@btangmu let's merge this PR (once. you're happy with other aspects) and I'll open a new one to fix the formatting for cldr-rdf.